frontend/test/__support__/e2e/helpers/e2e-dashboard-helpers.js

```diff
@@ -16,6 +16,7 @@ export function showDashboardCardActions(index = 0) {
 
 export function editDashboard() {
   cy.icon("pencil").click();
+  cy.findByText("You're editing this dashboard.");
```

Comment:
This can only improve the overall stability of tests using this particular helper.